src/schema/v2/previewSavedSearch.ts

```diff
@@ -102,49 +103,57 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(
         new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel))
       ),
-      resolve: async (_root, _args, { filterArtworksLoader }) => {
-        const suggestedFiltersByArtist: SearchCriteriaLabel[][] = await Promise.all(
-          _root.artistIDs.map((artistSlug) =>
-            getSuggestedFiltersByArtistSlug(artistSlug, filterArtworksLoader)
-          )
+      resolve: async ({ artistIDs }, _args, { filterArtworksLoader }) => {
+        if (!artistIDs) {
+          throw new Error("artistIDs are required to get suggested filters")
+        }
+
+        const gravityArgs = {
+          size: 0,
+          published: true,
```

Comment:
**question**: I don't see `published` as a supported query param on the [API endpoint](https://github.com/artsy/gravity/blob/main/app/api/v1/filter_endpoint.rb#L36-L37). Did you find that you needed this? I would expect that, by default, only published artworks are queried for.